src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -134,8 +132,10 @@ class MediaController(override val config: TableauxConfig,
       _ <- fileModel.deleteById(uuid)
       _ <- {
         import FutureUtils._
+
         promisify({ p: Promise[Unit] =>
-          vertx.fileSystem.delete(path.toString(), {result => p.success()})
+          // succeed even if file doesn't exist
+          vertx.fileSystem.delete(path.toString(), { result => p.success(()) })
```

Comment:
Permissions may be missing to delete it. Should it succeed or fail in that case?